tests/compiler/test_single.sh

```diff
@@ -1,6 +1,18 @@
 #!/usr/bin/env bash
 source ../common.sh
 
-compile_lean
+# First check the C version actually works...
+echo "running C program..."
+rm "./$f.out" || true
+compile_lean_c_backend
 exec_check "./$f.out"
 diff_produced
+
+# Then check the LLVM version
+if lean_has_llvm_support; then
+    echo "running LLVM program..."
+    rm "./$f.out" || true
+    compile_lean_llvm_backend
+    exec_check "./$f.out"
+    diff_produced
+fi
```

Comment:
This change is different from the other `.sh`s

Reply:
Fixed, the shell scripts ought to be consistent now.